instrumentation/graphql/lib/opentelemetry/instrumentation/graphql/tracers/graphql_tracer.rb

```diff
@@ -27,8 +27,17 @@ class GraphQLTracer < ::GraphQL::Tracing::PlatformTracing
           def platform_trace(platform_key, key, data)
             return yield if platform_key.nil?
 
-            tracer.in_span(platform_key, attributes: attributes_for(key, data)) do |_span|
-              yield
+            tracer.in_span(platform_key, attributes: attributes_for(key, data)) do |span|
+              yield.tap do |response|
+                if key == 'validate' && !response[:errors]&.empty?
+                  response[:errors].each do |error|
```

Comment:
If `response[:errors]` is `nil`, the `if` condition will be `true` and the `response[:errors].each` call will 💥
```
irb(main):001:0> x = {}
irb(main):002:0> !x[:foo]&.empty?
=> true
irb(main):003:0> x[:foo].each {|y| p y}
Traceback (most recent call last):
        4: from /opt/rubies/2.7.1/bin/irb:23:in `<main>'
        3: from /opt/rubies/2.7.1/bin/irb:23:in `load'
        2: from /opt/rubies/2.7.1/lib/ruby/gems/2.7.0/gems/irb-1.2.3/exe/irb:11:in `<top (required)>'
        1: from (irb):4
NoMethodError (undefined method `each' for nil:NilClass)
```

One option is to simply skip the test for `response[:errors]&.empty?` and use the safe navigation operator on the `response[:errors]&.each`.
```suggestion
                if key == 'validate'
                  response[:errors]&.each do |error|
```

However, see my next comment 😄